File: files/CSEC 26.05.py
import pandas as pd

input = pd.read_excel("files/Excel Challenge 26th May.xlsx", usecols="C:E", skiprows = 1, nrows = 18)
test  = pd.read_excel("files/Excel Challenge 26th May.xlsx", usecols="I:K", skiprows = 1, nrows = 5)

result = input.dropna().tail(5).reset_index(drop=True)
result["Amount"] = result["Amount"].astype("int64")
test.columns = input.columns

print(result.equals(test)) # True

File: files/2024-05-26/CSEC 26.05.R
library(tidyverse)
library(readxl)

input = read_excel("files/Excel Challenge 26th May.xlsx", range = "C2:E20")
test  = read_excel("files/Excel Challenge 26th May.xlsx", range = "I2:K7")

result = input %>%
  na.omit() %>%
  tail(5)

all.equal(result, test, check.attributes = FALSE)
# [1] TRUE